package/yast2-pam.changes

```diff
@@ -1,3 +1,9 @@
+-------------------------------------------------------------------
+Fri Jul 24 11:01:03 UTC 2020 - David Diaz <dgonzalez@suse.com>
+
+- Add a CFA based class to manage nsswitch.conf (bsc#1173119).
```

Comment:
well, change is not just adding it, but also using it, so main user change is support for configs in /usr/etc, right?